serverless/aws/resources/s3.py

```diff
@@ -43,15 +43,13 @@ def configure(self, service):
         super().configure(service)
 
         if service.has(Encryption):
-            self.resource.BucketEncryption = (
```

Comment:
Would have caused:

`Traceback (most recent call last):
  File "serverless.yml.py", line 92, in <module>
    service.resources.add(bucket)
  File "/Users/stan/.pyenv/versions/content-service/lib/python3.8/site-packages/serverless/service/resources.py", line 20, in add
    resource.configure(self._service)
  File "/Users/stan/.pyenv/versions/content-service/lib/python3.8/site-packages/serverless/aws/resources/s3.py", line 46, in configure
    self.resource.BucketEncryption = (
  File "/Users/stan/.pyenv/versions/content-service/lib/python3.8/site-packages/troposphere/__init__.py", line 229, in __setattr__
    self._raise_type(name, value, expected_type)
  File "/Users/stan/.pyenv/versions/content-service/lib/python3.8/site-packages/troposphere/__init__.py", line 245, in _raise_type
    raise TypeError(
TypeError: <class 'troposphere.s3.Bucket'>: Categories.BucketEncryption is <class 'tuple'>, expected <class 'troposphere.s3.BucketEncryption'>`